libraries/AP_Motors/AP_Motors_Thrust_Linearization.cpp

```diff
@@ -197,3 +203,25 @@ float Thrust_Linearization::get_compensation_gain() const
 #endif
     return ret;
 }
+
+// Check the voltage scale configuration
+bool Thrust_Linearization::battery_voltage_config_valid() const
+{
+    return (batt_voltage_max > 0) && (batt_voltage_min < batt_voltage_max);
+}
+
+bool Thrust_Linearization::arming_checks(size_t buflen, char *buffer) const
+{
+    if (get_spin_min() > 0.3) {
+        hal.util->snprintf(buffer, buflen, "%sSPIN_MIN too high %.2f > 0.3", AP_MOTORS_PARAM_PREFIX, get_spin_min());
+        return false;
+    }
+
+    if (is_positive(AP::battery().voltage(batt_idx)) && !battery_voltage_config_valid()) {
+        // If there is a valid voltage reading battery voltage scaling should be setup
+        hal.util->snprintf(buffer, buflen, "Set %sBAT_VOLT_MIN and %sBAT_VOLT_MAX", AP_MOTORS_PARAM_PREFIX, AP_MOTORS_PARAM_PREFIX);
```

Comment:
I wonder if the compiler is smart enough to 
```suggestion
        hal.util->snprintf(buffer, buflen, "Set " AP_MOTORS_PARAM_PREFIX " BAT_VOLT_MIN and " AP_MOTORS_PARAM_PREFIX "BAT_VOLT_MAX");
```